components/sensor/atm90e32.rst

```diff
@@ -44,6 +44,7 @@ The `CircuitSetup 6-Channel Energy Monitor <https://circuitsetup.us/index.php/pr
     sensor:
       - platform: atm90e32
         cs_pin: 5
+	write:
```

Comment:
?

Reply:
cutting error. fixed  
The write example was move to another section